docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.6"
+
+services:
+  peripheral-manager-ethernet:
+    build: .
+    restart: on-failure
+    network: host
```

Comment:
have you tested this? I'm almost sure the code won't work in this network

Reply:
You are right. Needs to be `network_mode: host`. It has been corrected